.circleci/main.yml

```diff
@@ -2269,3 +2446,28 @@ commands:
             wget -O ./bazel "https://github.com/bazelbuild/bazelisk/releases/download/v1.19.0/bazelisk-linux-amd64"
             chmod +x ./bazel
             sudo mv ./bazel /home/circleci/bin/bazel
+  node-install-all:
+    steps:
+      - run:
+          name: Npm install project root, frontend and backend
+          command: npm ci --prefix ./console && npm ci --prefix ./console/backend && npm ci --prefix ./console/frontend
+  node-install-frontend-backend:
+    steps:
+      - run:
+          name: Npm install frontend and backend
+          command: npm ci --prefix ./console/backend && npm ci --prefix ./console/frontend
+  install-console-e2e-deps:
+    steps:
+      - attach_workspace:
+          at: ./
+      - run:
+          name: Install kubectl
+          command: bazel run --config=remotecache --remote_header=x-buildbuddy-api-key=${BUILDBUDDY_API_KEY} //tools/kubectl:install
+      - run:
+          name: Install pachctl
+          command: bazel run --config=remotecache --remote_header=x-buildbuddy-api-key=${BUILDBUDDY_API_KEY} //src/server/cmd/pachctl:install
+      - run:
+          name: Setup k8s cluster
+          command: |
+            export CI=false # pachdev bug; CI=true assumes docker executor, not machine executor
+            bazel run --config=remotecache --remote_header=x-buildbuddy-api-key=${BUILDBUDDY_API_KEY} //src/testing/pachdev -- create-cluster local --starting-port=30100
```

Comment:
This setup step is copied from the jupyter e2e tests. They are able to deploy pachyderm in two namespaces (one with auth and one without) in the same cluster. We can not use the two namespace approach without some reconfiguration in our tests so I deploy pachyderm in each of the jobs, without auth for the community edition tests and with auth for our authenticated test suite.